site/content/en/docs/Tasks/_index.md

```diff
@@ -1,11 +1,10 @@
-
```

Comment:
Do we need this file? If we do, we should probably delete the boilerplate.

Reply:
We do need it. More coming soon.